src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -60,7 +62,7 @@ void SendLocationRequest::Run() {
     return;
   }
 
-  const smart_objects::SmartObject& msg_params = (*message_)[strings::msg_params];
+  SmartObject& msg_params = (*message_)[strings::msg_params];
```

Comment:
@dev-gh  before my changes for functions that was need not const msg_params used *(message_)[strings::msg_params], I thins that not const variable is better that using hack for hiding usage of not const msg_params